whitepaper/kubernetes-policy-management-whitepaper.md

```diff
@@ -18,18 +18,20 @@
   - [Configurations](#configurations)
   - [Runtime Behaviors](#runtime-behaviors)
 - [Policy Architecture](#policy-architecture)
-  - [Policy Definition Language (PDL)](#policy-definition-language-(pdl))
-  - [Policy Authoring Point (PAP)](#policy-authoring-point-(pap))
-  - [Policy Management Point (PMP)](#policy-management-point-(pmp))
-  - [Policy Enforcement Point (PEP)](#policy-enforcement-point-(pep))
-  - [Policy Reporting (PR)](#policy-reporting-(pr))
+  - [Policy Definition Language (PDL)](#policy-definition-language-(pdl)) This represents the syntax of how policies are represented.
+  - [Policy Administration Point (PAP)](#policy-authoring-point-(pap)) This is where the policy is specified. Typically it would be using a command line interface, a graphical user interface, or GiTOps. The preferred approach is GitOps because it 
+  s policies to be treated just like source code and allows proper access control and auditability on specification and updates of policies.
+  - [Policy Management Point (PMP)](#policy-management-point-(pmp)) This represents the central management Hub that distributes policy to the fleet of desired clusters, consolidates policy violations across fleet, integrates with enterprise tools (security operations center, incident management, GRC etc), automates remediation, provides security and compliance readiness posture, and renders contextual posture for SRE, application developer personas. 
+  - [Policy 
+  Point (PEP)](#policy-enforcement-point-(pep))Returns results of policy evaluation in ‘inform’ mode, enforces control to desired state in ‘enforce’ mode, optionally invokes a Policy Decision Point (PDP) when the PEP is distinct from the policy evaluation engine.
+  - [Policy Reporting (PR)](#policy-reporting-(pr)) This represents how results of evaluating a policy against the state of a control are represented. Standardizing this aspect allow results to be processed and analyzed in a consistent manner and provides ease of integration when various pieces of the policy architecture are delivered by different vendors.
 - [Policy Lifecycle](#policy-lifecycle)
-  - [Authoring](#authoring)
-  - [Versioning](#versioning)
-  - [Deployment](#deployment)
-  - [Enforcement](#enforcement)
-  - [Detection](#detection)
-  - [Remediation](#remediation)
+  - [Authoring](#authoring)Policies are authored at the PAP.
+  - [Versioning](#versioning) Versioning of policies can be easily accomplished when using GitOps approach as policies are treated just like source code and manintained in GitHub.
+  - [Deployment](#deployment)Policies are deployed by the PMP by binding them to the managed clusters. A flexible approach of doing so is by associating labels with the clusters and applying policies to clusters that meet certain placement rules derived from these labels.
```

Comment:
"binding" meaning creating an api call which creates a resource instance (record in etcd), correct? as to labels - the "applying policies" based on label happens via some operator or sidecar, eg OPA, kyverno yes?